app/account/views.py

```diff
@@ -250,3 +250,26 @@ def unconfirmed():
     if current_user.is_anonymous() or current_user.confirmed:
         return redirect(url_for('main.index'))
     return render_template('account/unconfirmed.html')
+
+
+@account.route('/profile/<int:user_id>')
+@login_required
+def profile(user_id):
+    """Display a user's profile"""
+    user = User.query.get(int(user_id))
+    if user is None:
+        return render_template('errors/404.html')
+    if user.id == current_user.id:
+        isCurrent = True
+    else:
+        isCurrent = False
+    role = Role.query.get(int(user.role_id))
+    return render_template('account/profile.html', user=user,
+                           role=role, isCurrent=isCurrent)
+
+
+@account.route('/profile/edit')
+@login_required
+def edit():
```

Comment:
let's give this function a more descriptive name like `edit_profile()` or `profile_edit()`